buyer-api/src/operations/addDataResponse.js

```diff
@@ -20,23 +20,24 @@ import { addProcessDataResponseJob } from '../queues/dataResponseQueue';
 export const addDataResponse = async (dataOrder, dataResponse) => {
   const { status: st, notariesAddresses } = dataOrder;
   if (st !== 'created') {
-    return { error: 'Can\'t accept DataReponse' };
+    return { error: { message: "Can't accept DataReponse", status: 'closed' } };
   }
 
   const {
-    orderId,
-    sellerAddress,
-    encryptedData,
-    notaryAddress,
-    ...rest
+    orderId, sellerAddress, encryptedData, notaryAddress, ...rest
   } = dataResponse;
 
   const sellerId = await sellers.safeFetch(sellerAddress, 0);
 
   const id = `${orderId}:${sellerAddress}`;
 
   if (!notariesAddresses.includes(notaryAddress)) {
-    return { error: `Can't accept DataReponse for notary ${notaryAddress}` };
+    return {
+      error: {
+        message: `Can't accept DataReponse for notary ${notaryAddress}`,
+        status: 'unprocessable',
+      },
```

Comment:
Should we consider using something similar to what we have in the mobile app? i.e.: an `error.js` file that has all the returned errors, I'm saying this because errors and status codes are part of our documentation. So maybe we could have something like 
```javascript
export const ERROR_CLOSED_DATA_RESPONSE = {
message: "Can't accept DataReponse", 
statusCode: 422
}
```

And then in the API we just "[boomify](https://github.com/hapijs/boom/blob/master/API.md#boomifyerr-options)" the error.
What do you guys think? @Meketh @nicoayala

Reply:
I know it is handy to have operations responding a structure like that one, but since operations can be called from different "parts of the system" (like API endpoint handlers, Background Jobs, Tasks, Blockchain event handlers, etc.) and these don't necessary share the same codes as the http protocol, I'd go with something custom, agnostic of the protocol used to transmit the error. 

What about:
```js
export const ERROR_ADD_DATA_RESPONSE_INVALID_NOTARY = {
  message: "Can't accept DataReponse", 
  code: 'add_data_response.invalid_notary' // 'add_data_response.closed_data_order' for the case above
}
```
As a result, the response for this operation contains an error message and a code that indicates what happened with execution of this operation. The client, which in this case is the express handler, checks for the error, and responds a 422 with the error content.

I agree in that we have to standardize the operations interface as fast as we can.

Reply:
I like it, as long as we have the error clearly defined somewhere, and we can check with a "static code", in this case `add_data_response.invalid_notary`, I think it will work.